vite.config.ts

```diff
@@ -7,16 +7,23 @@ export default defineConfig(({mode}) => {
   const env = loadEnv(mode, process.cwd(), '');
   return {
     define: {
+      'process.env.DEVCON6_SESSSIONS_HASH': JSON.stringify(env.DEVCON6_SESSSIONS_HASH) ?? JSON.stringify(''),
       'process.env.BEE_API_URL': JSON.stringify(env.BEE_API_URL) ?? JSON.stringify('http://localhost:1633/'),
-      'process.env.FEED_OWNER_ADDRESS': JSON.stringify(env.FEED_OWNER_ADDRESS) ?? JSON.stringify('5F7b9cc47803fD4586E92c1702b114B22DB5dd4f'),
-      'process.env.HEALTH_CHECK_DATA_REF': JSON.stringify(env.HEALTH_CHECK_DATA_REF) ?? JSON.stringify('2fdbf84ba761d65f3b848f22cf3c0f6899c61ada4347b9a1b3fcf0ea27dde6b3')
+      'process.versions': JSON.stringify({ node: 'browser-mock' }), // Mocking process.versions.node
+      'process.nextTick': (callback: Function) => setTimeout(callback, 0),
```

Comment:
`npm run build` fails with:
```
error during build:
[vite:define] Transform failed with 1 error:
node_modules/esbuild/lib/main.js:295:10: ERROR: Expected value for define "process.nextTick" to be a string, got undefined instead
```

Reply:
I removed process.nextTick.

This is related to `pino` and `prettier`. Later I solved it by commenting out that part in the lib itself, so it is not using command line colorization. So this line is not needed. Now it should work.